examples/inputs/example_01d/buses.csv

```diff
@@ -1,3 +1,3 @@
-name,v_nom,carrier,x,y
-north,380,AC,9.598,53.5585
-south,380,AC,13.607,51.0769
+name,v_nom,carrier,x,y,shape
```

Comment:
I really really doubt users would input directly geopandas shapes in here, We should work with coordinates like x y or lat long and convert it inside code ourselves